src/index.js

```diff
@@ -13,5 +15,23 @@ function render() {
 		.map(result => `<tr><td>${fuzzysort.highlight(result)}</td><td>${result.obj.version}</td></tr>`)
 		.join("");
 }
+
+function updateQueryString() {
+	queryString.set('q', input.value);
+
+	const newLocation = `${window.location.pathname}?${queryString.toString()}`;
+	history.pushState(null, '', newLocation);
```

Comment:
What do you think about using `History.replaceState()` instead so the history does not clog up from pushing a new history entry on every keystroke?

Reply:
Good idea, thanks, I'll change it to `History.replaceState()`.